docs/source/advanced/model_parallel.rst

```diff
@@ -718,6 +718,73 @@ DDP Optimizations
 ^^^^^^^^^^^^^^^^^
 
 
+When Using DDP Strategies, Set find_unused_parameters=False
```

Comment:
This is unfortunately using the same title level as the super header for "DDP Optimizations".